images/builder/install-protoplugins.sh

```diff
@@ -8,8 +8,8 @@ set -o errexit
 set -o pipefail
 set -o nounset
 
-# c285fc70e095eccc98d79b9a133e1e328141aefd == tag: cmd/protoc-gen-go-grpc/v1.2.0
-go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@c285fc70e095eccc98d79b9a133e1e328141aefd
-go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.28.0
+# 8ba23be9613c672d40ae261d2a1335d639bdd59b == tag: cmd/protoc-gen-go-grpc/v1.3.0
```

Comment:
We probably could have those be updated by Renovate going forward. It would need a custom regex manager, but it's possible to match git tags to git shas and have Renovate update the sha if the tag changes.